src/provider/proxyprovider.cpp

```diff
@@ -0,0 +1,86 @@
+#include "provider/proxyprovider.h"
+#include "apiresponse.h"
+#include "httpwrapper.h"
+
+namespace internal
+{
+    bool isRequestSuccessful(ErdGenericApiResponse const &response)
```

Comment:
Not sure, why isn't this a member function (improve cohesion)?

Reply:
I've removed that function from internal namespace anyway.
However, there were two reasons for this:

1. improve cohesion -> YES. That function did not use any internal member function, was rather a helper
2. **The most important reason:** I did not want to include`apiresponse.h` in `proxyprovider.h`. This way, it is only used in `proxyprovider.cpp`, so CPP SDK users don't have to include internal files for their APIs (`apiresponse.h`)